### PR TITLE
Enable portal to redeem invites

### PR DESCRIPTION
**Pull request by jasonpaige**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by LaurenceRawlings — APPROVED** [review]

**Comment by lucas-phillips28** [comment]
Need to add endpoint to list of not authorised endpoints in `SecurityConfig`

**Review by lucas-phillips28 — CHANGES_REQUESTED** [review]
Need to add endpoint to list of non-authorised endpoints in SecurityConfig

File: src/main/java/uk/gov/hmcts/reform/preapi/config/SecurityConfig.java [hunk 1]
```diff
@@ -32,7 +32,9 @@ public class SecurityConfig {
         "/users/by-email/**",
         "/reports/**",
         "/audit/**",
-        "/error"
+        "/error",
+        "/invites",
```
Comment (lucas-phillips28) [inline src/main/java/uk/gov/hmcts/reform/preapi/config/SecurityConfig.java:36]: Why do we need this one?

Comment (jasonpaige) [reply src/main/java/uk/gov/hmcts/reform/preapi/config/SecurityConfig.java:36]: yeh, I need to check if the user is new by calling GET /invites?email=someone@somewhere.com and if i get a response I'll POST /invites/redeem/someone@somewhere.com

Would that need to be /invites/redeem/** in the SecurityConfig actually or do params not count?

Comment (lucas-phillips28) [reply src/main/java/uk/gov/hmcts/reform/preapi/config/SecurityConfig.java:36]: Just tested to make sure- it does ignore the params

**Review by lucas-phillips28 — APPROVED** [review]

**Review by LaurenceRawlings — APPROVED** [review]

**Pull request merged**